question-answering-using-openai/main.bal

```diff
@@ -0,0 +1,211 @@
+import ballerina/http;
+import ballerinax/googleapis.sheets;
+import ballerinax/openai;
+import ballerina/lang.'float as langFloat;
```

Comment:
No need to import this lib, instead use `.` to invoke functions.

Reply:
addressed in [a3d99a2](https://github.com/ballerina-guides/ai-samples/pull/2/commits/a3d99a29fc85ec81cafb7f92df253d5cbe4b6449)